Homework_7.py

```diff
@@ -0,0 +1,95 @@
+# Homework 7
+
+import csv  # import module for work with csv
+import os  # import module for ability to delete files
+from Homework_6 import *  # import previous homeworks (5-6)
+
+
+class CountWords:
+    def parse_file(self, list_with_words):
+        with open('posts.txt', 'r') as source_file:  # open file with posts
+            reader = csv.reader(source_file)  # read each line of the file
+            for row in reader:
+                for element in row:
+                    if element != []:
+                        split_element = element.split(' ')
+                        for word in split_element:
+                            if word != '':
+                                list_with_words.append(word.lower())
+        return list_with_words
+
+    def count_number(self, dict_for_words, list_with_words):
```

Comment:
Same thing, you can just create empty dict in the beginning of the function instead of every time pass empty dict as a parameter to this function.